openhtf/io/frontend/__main__.py

```diff
@@ -225,40 +214,65 @@ def post(self, host, port, prompt_id):
     msg = json.JSONEncoder().encode(
         {'id': prompt_id, 'response': self.request.body})
     self.store[host, port].Notify(msg)
-    self.write('SENT')
+    self.write('')
 
 
 def main(argv):
   """Start the frontend."""
-  try:
-    argv = FLAGS(argv)  # parse flags
-  except gflags.FlagsError, e:
-    print('%s\nUsage: %s ARGS\n%s' % (e, sys.argv[0], FLAGS))
-    sys.exit(1)
+  parser = argparse.ArgumentParser(description='OpenHTF web frontend server.',
+                                   prog='python -m openhtf.io.frontend')
+  parser.add_argument('--port', type=int, default=12000,
+                      help='Port on which to serve the frontend.')
+  parser.add_argument('--poll_interval_ms', type=int, default=3000,
+                      help='Time between frontend polls in milliseconds.')
+  parser.add_argument('--discovery_interval_s', type=int, default=5,
```

Comment:
Oh yeah! I was thinking about that a while back, just forgot to implement it here. It would be really good to have. I'll add it.